src/web_ui/templates/base.html

```diff
@@ -13,6 +13,9 @@
     <link rel="stylesheet" href="{{ url_for('static', filename='css/bootstrap.min.css') }}">
   </head>
   <body>
+    {% if request.path != '/' %}
+      <a href="/"> Back to home page</a>
```

Comment:
This is so great - I always want to include this!

(but why do we have a space between the open tag and `Back`?)

Reply:
Thanks, nice spot! Originally I wanted to add an `arrow_back` in between the tag and `Back` and when I removed it I forgot the space